assembly/pom.xml

```diff
@@ -302,6 +303,51 @@
 			<artifactId>transport</artifactId>
 			<version>${elasticsearch-client-transport.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.plugin</groupId>
+			<artifactId>transport-netty3-client</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.plugin</groupId>
+			<artifactId>transport-netty4-client</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.plugin</groupId>
+			<artifactId>reindex-client</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.plugin</groupId>
+			<artifactId>lang-mustache-client</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.plugin</groupId>
+			<artifactId>percolator-client</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-api</artifactId>
+			<version>2.7</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+			<version>2.7</version>
+		</dependency>
+		<dependency>
+		    <groupId>io.netty</groupId>
+		    <artifactId>netty-all</artifactId>
+		    <version>4.1.5.Final</version>
+		</dependency>
+		<dependency>
+		    <groupId>io.netty</groupId>
+		    <artifactId>netty</artifactId>
+		    <version>3.10.6.Final</version>
+		</dependency>
```

Comment:
BTW I get the same when I try to access the data from the console
`1) Error injecting constructor, org.eclipse.kapua.service.datastore.client.ClientUnavailableException: Error: Cannot load the provided client class name [org.eclipse.kapua.service.datastore.client.transport.TransportDatastoreClient]. Check the configuration.
  at org.eclipse.kapua.service.datastore.internal.ClientInfoRegistryServiceImpl.<init>(ClientInfoRegistryServiceImpl.java:84)
  while locating org.eclipse.kapua.service.datastore.internal.ClientInfoRegistryServiceImpl
  at org.eclipse.kapua.locator.guice.KapuaModule.configure(KapuaModule.java:99)
  while locating org.eclipse.kapua.service.datastore.ClientInfoRegistryService

1 error
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1028)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1054)
	at org.eclipse.kapua.locator.guice.GuiceLocatorImpl.getService(GuiceLocatorImpl.java:56)
	at org.eclipse.kapua.service.datastore.internal.mediator.DatastoreMediator.<clinit>(DatastoreMediator.java:68)
	... 85 common frames omitted
Caused by: org.eclipse.kapua.service.datastore.client.ClientUnavailableException: Error: Cannot load the provided client class name [org.eclipse.kapua.service.datastore.client.transport.TransportDatastoreClient]. Check the configuration.
	at org.eclipse.kapua.service.datastore.internal.client.DatastoreClientFactory.getInstance(DatastoreClientFactory.java:72)
	at org.eclipse.kapua.service.datastore.internal.ClientInfoRegistryFacade.<init>(ClientInfoRegistryFacade.java:71)
	at org.eclipse.kapua.service.datastore.internal.ClientInfoRegistryServiceImpl.<init>(ClientInfoRegistryServiceImpl.java:95)
	at org.eclipse.kapua.service.datastore.internal.ClientInfoRegistryServiceImpl$$FastClassByGuice$$e97ca0d7.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:56)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:194)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
	at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1019)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1015)
	... 88 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.kapua.service.datastore.internal.client.DatastoreClientFactory.getInstance(DatastoreClientFactory.java:68)
	... 104 common frames omitted
Caused by: org.eclipse.kapua.service.datastore.client.ClientUnavailableException: Error: Cannot load the provided client class name [org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider]. Check the configuration.
	at org.eclipse.kapua.service.datastore.client.transport.TransportDatastoreClient.<init>(TransportDatastoreClient.java:153)
	at org.eclipse.kapua.service.datastore.client.transport.TransportDatastoreClient.getInstance(TransportDatastoreClient.java:117)
	... 109 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.kapua.service.datastore.client.transport.TransportDatastoreClient.<init>(TransportDatastoreClient.java:148)
	... 110 common frames omitted
Caused by: org.eclipse.kapua.service.datastore.client.ClientUnavailableException: Error: Failed to configure ElasticSearch transport
	at org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider.<init>(EsTransportClientProvider.java:198)
	at org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider.<init>(EsTransportClientProvider.java:174)
	at org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider.<init>(EsTransportClientProvider.java:164)
	at org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider.init(EsTransportClientProvider.java:91)
	... 115 common frames omitted
Caused by: java.lang.NoClassDefFoundError: org/apache/logging/log4j/Logger
	at org.elasticsearch.common.logging.Loggers.getLogger(Loggers.java:101)
	at org.elasticsearch.common.xcontent.support.AbstractXContentParser.<init>(AbstractXContentParser.java:57)
	at org.elasticsearch.common.xcontent.json.JsonXContentParser.<init>(JsonXContentParser.java:44)
	at org.elasticsearch.common.xcontent.json.JsonXContent.createParser(JsonXContent.java:103)
	at org.elasticsearch.common.settings.Setting.parseableStringToList(Setting.java:832)
	at org.elasticsearch.common.settings.Setting.lambda$listSetting$27(Setting.java:786)
	at org.elasticsearch.common.settings.Setting.listSetting(Setting.java:791)
	at org.elasticsearch.common.settings.Setting.listSetting(Setting.java:786)
	at org.elasticsearch.common.network.NetworkService.<clinit>(NetworkService.java:50)
	at org.elasticsearch.client.transport.TransportClient.newPluginService(TransportClient.java:98)
	at org.elasticsearch.client.transport.TransportClient.buildTemplate(TransportClient.java:126)
	at org.elasticsearch.client.transport.TransportClient.<init>(TransportClient.java:268)
	at org.elasticsearch.transport.client.PreBuiltTransportClient.<init>(PreBuiltTransportClient.java:125)
	at org.elasticsearch.transport.client.PreBuiltTransportClient.<init>(PreBuiltTransportClient.java:111)
	at org.elasticsearch.transport.client.PreBuiltTransportClient.<init>(PreBuiltTransportClient.java:101)
	at org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider.<init>(EsTransportClientProvider.java:191)
	... 118 common frames omitted
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.Logger
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.eclipse.jetty.webapp.WebAppClassLoader.loadClass(WebAppClassLoader.java:560)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 134 common frames omitted
`